package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "biome lint . --diagnostic-level=error",
     "new": "modern new",
     "pre": "modern pre",
-    "prepare": "pnpm run build -p @rsdoctor/* && husky install",
```

Comment:
Why need delete this parameter ?

Reply:
Use the build script from this package.json

Reply:
`pnpm run build`  is the same as `cross-env NX_DAEMON=false nx run-many -t build --exclude @examples/* @scripts/* @e2e/* --parallel=10`

`pnpm run build -p "@rsdoctor/*"`   is the same as running build scripts under `@rsdoctor/*` projects

Reply:
Sorry, I misunderstood.
`pnpm run build -p "@rsdoctor/*"` is the same as ` cross-env NX_DAEMON=false nx run-many -t build --exclude @examples/* @scripts/* @e2e/* --parallel=10 "-p" "@rsdoctor/*"` 

Now that it has been merged and it's fine and harmless 😸